code/modules/food_and_drinks/recipes/tablecraft/recipes_meat.dm

```diff
@@ -81,6 +81,37 @@
 	result = /obj/item/reagent_containers/food/snacks/sashimi
 	subcategory = CAT_MEAT
 
+/datum/crafting_recipe/food/sushie_basic
+	name = "Funa Hosomaki"
+	reqs = list(
+		/datum/reagent/consumable/soysauce = 3,
+		/obj/item/reagent_containers/food/snacks/salad/ricebowl = 1,
+		/obj/item/reagent_containers/food/snacks/carpmeat = 2,
+		/obj/item/reagent_containers/food/snacks/grown/kudzupod = 3,
+	)
+	result = /obj/item/reagent_containers/food/snacks/sushie_basic
+	subcategory = CAT_MEAT
+
+/datum/crafting_recipe/food/sushie_adv
+	name = "Funa Nigiri"
+	reqs = list(
+		/datum/reagent/consumable/soysauce = 5,
+		/obj/item/reagent_containers/food/snacks/salad/ricebowl = 1,
+		/obj/item/reagent_containers/food/snacks/carpmeat = 1
+	)
+	result = /obj/item/reagent_containers/food/snacks/sushie_adv
+	subcategory = CAT_MEAT
+
+/datum/crafting_recipe/food/sushie_pro
+	name = "Well made Funa Nigiri"
```

Comment:
not capitalized correctly

Reply:
why does this differentiation even exist?

Reply:
Its capitalized as others, fits with the file and other formatting.
Two are for more options, and such.

Reply:
Literally no reason.

Reply:
Its capitalized as others, fits with the file and other formatting.
Two are for more options, and such.